### PR TITLE
Filepath fixes

### PR DESCRIPTION
**Pull request by InAJacket**
Renamed all files and folders to fit the style guide. Game should now work on Linux without hiccups.

Also added a minimal CONTRIBUTING.md file.

### CODE REVIEW

File: addons/sliderlabel/SliderLabel.gd
Comment (noah1510) [inline addons/sliderlabel/SliderLabel.gd:1]: rename the file to slider_label to be safe

File: addons/sliderlabel/SliderLabel.svg
Comment (noah1510) [inline addons/sliderlabel/SliderLabel.svg:1]: also should be slider_label

File: environment/Materials/Grass.material.depren
Comment (noah1510) [inline environment/Materials/Grass.material.depren:1]: should be grass

**Review by noah1510 — CHANGES_REQUESTED** [review]
A few files are not renamed yet.
Better do it now than later

**Review by noah1510 — APPROVED** [review]
Looks good to me now

**Comment by InAJacket** [comment]
Ok that should be everything

**Comment by noah1510** [comment]
I manually merged it into upstream.
I am closing this now

**Pull request closed without merging**